From: Kotty <kotty@guns-n-girls.de>
Date: Tue, 9 Feb 2016 18:06:17 +0100
Subject: kfreebsd-makefile.patch

---
 configure.server | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/configure.server b/configure.server
index ff7803f..8662fc9 100755
--- a/configure.server
+++ b/configure.server
@@ -533,16 +533,17 @@ echo "# Large File Support settings"     >>Makefile
 echo "LFSDEF = $LFS"                     >>Makefile
 
 echo "" >>Makefile
-if test -r build/Makefile.`uname -s`
+if test -r build/Makefile.`uname -s | tr '/' '_'`
 then
-	echo "include build/Makefile.`uname -s`" >>Makefile
+	# use sinclude here so "clean" works even when build/Makefile.GNU_kFreeBSD is not patched yet
+	echo "sinclude build/Makefile.`uname -s | tr '/' '_'`" >>Makefile
 	echo ""
-	echo "Using `uname -s` Makefile settings"
+	echo "Using `uname -s | tr '/' '_'` Makefile settings"
 	echo ""
 else
 	echo "include build/Makefile.generic" >>Makefile
 	echo ""
-	echo "Using GENERIC Makefile settings"
+	echo "Using GENERIC Makefile settings (uname -s is `uname -s`)"
 	echo ""
 	echo "If this fails, change the compile settings in Makefile"
 	echo ""
