From: Christoph Berg <myon@debian.org>
Date: Tue, 26 Jan 2016 16:31:42 +0100
Subject: 63_netstat-ant-vs-ipv6-address-truncating

---
 client/xymonclient-linux.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/xymonclient-linux.sh b/client/xymonclient-linux.sh
index 235de11..a6f7822 100755
--- a/client/xymonclient-linux.sh
+++ b/client/xymonclient-linux.sh
@@ -73,7 +73,7 @@ echo "[netstat]"
 netstat -s
 echo "[ports]"
 # Bug in RedHat's netstat spews annoying error messages. 
-netstat -antu 2>/dev/null
+netstat -antuW 2>/dev/null
 echo "[ifstat]"
 /sbin/ifconfig 2>/dev/null
 # Report mdstat data if it exists
