From: Kotty <kotty@guns-n-girls.de>
Date: Tue, 9 Feb 2016 18:07:20 +0100
Subject: disable_SSLv3.patch

---
 xymonnet/contest.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/xymonnet/contest.c b/xymonnet/contest.c
index b4e2a9a..884ad0c 100644
--- a/xymonnet/contest.c
+++ b/xymonnet/contest.c
@@ -482,8 +482,6 @@ static void setup_ssl(tcptest_t *item)
 		  case SSLVERSION_V2:
 			item->sslctx = SSL_CTX_new(SSLv2_client_method()); break;
 #endif
-		  case SSLVERSION_V3:
-			item->sslctx = SSL_CTX_new(SSLv3_client_method()); break;
 		  case SSLVERSION_TLS1:
 			item->sslctx = SSL_CTX_new(TLSv1_client_method()); break;
 		  default:
