From: Christoph Berg <myon@debian.org>
Date: Tue, 26 Jan 2016 16:31:42 +0100
Subject: 81_confreport-segfault

---
 web/confreport.c | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/web/confreport.c b/web/confreport.c
index 4c8a0b2..f6b29ae 100644
--- a/web/confreport.c
+++ b/web/confreport.c
@@ -288,9 +288,10 @@ static void print_host(hostlist_t *host, htnames_t *testnames[], int testcount)
 		}
 		else if (is_net_test(itm)) {
 			colname = strdup(itm);
+			visdata = strdup("");
 		}
 
-
+		if (!visdata) visdata = strdup("");
 		if (colname) {
 			tag_t *newitem;
 
@@ -307,7 +308,9 @@ addtolist:
 			}
 			else {
 				/* Multiple tags for one column - must be http */
-				newitem->visualdata = (char *)realloc(newitem->visualdata, strlen(newitem->visualdata) + strlen(visdata) + 5);
+				newitem->visualdata = newitem->visualdata ?
+					(char *)realloc(newitem->visualdata, strlen(newitem->visualdata) + strlen(visdata) + 5) :
+					(char *)malloc(strlen(visdata) + 5);
 				strcat(newitem->visualdata, "<br>");
 				strcat(newitem->visualdata, visdata);
 			}
