From: Kotty <kotty@guns-n-girls.de>
Date: Tue, 9 Feb 2016 18:08:31 +0100
Subject: Hostname-refers-now-to-findhost.sh-so-we-can-jump-to.patch

---
 xymond/webfiles/hostsvc_header | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xymond/webfiles/hostsvc_header b/xymond/webfiles/hostsvc_header
index b562373..c497c35 100644
--- a/xymond/webfiles/hostsvc_header
+++ b/xymond/webfiles/hostsvc_header
@@ -30,7 +30,7 @@
     <FONT FACE="Arial, Helvetica" SIZE="+1" COLOR="silver"><B>&XYMONLOGO</B></FONT>
   </TD>
   <TD VALIGN=MIDDLE ALIGN=CENTER WIDTH="40%">
-    <CENTER><FONT FACE="Arial, Helvetica" SIZE="+1" COLOR="silver"><B>&XYMWEBHOST - &XYMWEBSVC</B></FONT></CENTER>
+    <CENTER><FONT FACE="Arial, Helvetica" SIZE="+1" COLOR="silver"><B><a href="findhost.sh?host=&XYMWEBHOST&jump=1">&XYMWEBHOST</a> - &XYMWEBSVC</B></FONT></CENTER>
   </TD>
   <TD VALIGN=MIDDLE ALIGN=RIGHT WIDTH="30%">
    <FONT FACE="Arial, Helvetica" SIZE="+1" COLOR="silver"><B>&XYMWEBDATE</B></FONT>
